tests/core/test_common.py

```diff
@@ -119,6 +119,7 @@ def test_name(self):
 
         self.assertEqual(self.ac.normalize("PauseTotalNs", "prefix", fix_case = True), "prefix.pause_total_ns")
         self.assertEqual(self.ac.normalize("Metric.wordThatShouldBeSeparated", "prefix", fix_case = True), "prefix.metric.word_that_should_be_separated")
+        self.assertEqual(self.ac.normalize_device_name("//device@name"), "//device_name")
```

Comment:
Since `normalize_device_name`  replace a lot more than just `@` we should test them also.

something like:
```python
self.assertEqual(self.ac.normalize_device_name(",@+*-()[]{}//device@name"), "___________//device_name")
```